diff -dur meka-r435.orig/srcs/app_cheatfinder.c meka-r435/srcs/app_cheatfinder.c
--- meka-r435.orig/srcs/app_cheatfinder.c	2013-07-23 23:46:54.177599000 +0200
+++ meka-r435/srcs/app_cheatfinder.c	2013-08-18 14:46:55.422806619 +0200
@@ -568,7 +568,7 @@
 static void CheatFinder_CallbackComparer(t_widget* w)
 {
 	t_cheat_finder* app = (t_cheat_finder*)w->box->user_data;
-	CheatFinder_ReduceMatches(app, (t_cheat_finder_comparer)(int)w->user_data);
+	CheatFinder_ReduceMatches(app, (t_cheat_finder_comparer)(intptr_t)w->user_data);
 }
 
 static void CheatFinder_CallbackReduce(t_widget* w)
