From 8c66287600aecc82a9ff897d497ca2d0ae864069 Mon Sep 17 00:00:00 2001
From: ocornut <omar@miracleworld.net>
Date: Wed, 7 Jan 2026 16:19:11 +0100
Subject: [PATCH] SDSC console: avoid using MSVC specific functions (#139, #35)

Amend 66e8272540
---
 meka/srcs/sdsc.cpp | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/meka/srcs/sdsc.cpp b/meka/srcs/sdsc.cpp
index 1aa14d4d..3becfbe3 100644
--- a/meka/srcs/sdsc.cpp
+++ b/meka/srcs/sdsc.cpp
@@ -250,7 +250,7 @@ bool SDSC_Format_Error(const char* format, ...)
     // Print the message over the format string
     va_list params;
     va_start(params, format);
-    const int chars_added = vsprintf_s(p_format_start, sdsc_console_buffer_size - (p_format_start - sdsc_console_buffer), format, params);
+    const int chars_added = vsnprintf(p_format_start, sdsc_console_buffer_size - (p_format_start - sdsc_console_buffer), format, params);
     va_end(params);
     // Point to the following char
     p_next_char = p_format_start + chars_added;
@@ -574,19 +574,19 @@ void SDSC_Try_Format()
     {
     case 'd':
         // Signed decimal
-        formatted_length = sprintf_s(buffer, buffer_size, "%d", is_byte ? (s8)data : (s16)data);
+        formatted_length = snprintf(buffer, buffer_size, "%d", is_byte ? (s8)data : (s16)data);
         break;
     case 'u':
         // Unsigned decimal
-        formatted_length = sprintf_s(buffer, buffer_size, "%u", is_byte ? (u8)data : data);
+        formatted_length = snprintf(buffer, buffer_size, "%u", is_byte ? (u8)data : data);
         break;
     case 'x':
         // Lowercase hex
-        formatted_length = sprintf_s(buffer, buffer_size, "%x", is_byte ? (u8)data : data);
+        formatted_length = snprintf(buffer, buffer_size, "%x", is_byte ? (u8)data : data);
         break;
     case 'X':
         // Uppercase hex
-        formatted_length = sprintf_s(buffer, buffer_size, "%X", is_byte ? (u8)data : data);
+        formatted_length = snprintf(buffer, buffer_size, "%X", is_byte ? (u8)data : data);
         break;
     case 'b':
         // Binary
@@ -649,7 +649,7 @@ void SDSC_Try_Format()
     }
 
     // Replace the format string with the result
-    strncpy_s(p_format_start, sdsc_console_buffer_size - (p_format_start - sdsc_console_buffer), p, width);
+    snprintf(p_format_start, sdsc_console_buffer_size - (p_format_start - sdsc_console_buffer), p, width);
     p_next_char = p_format_start + formatted_length;
     // And signal that we are done
     p_format_start = NULL;
