diff -dur meka.orig/srcs/eagle.asm meka/srcs/eagle.asm
--- meka.orig/srcs/eagle.asm	2007-07-29 20:58:28.000000000 +0200
+++ meka/srcs/eagle.asm	2008-08-22 17:39:10.000000000 +0200
@@ -1523,3 +1523,7 @@
 _tripline3		resb 8
 _dlx			resb 32
 
+%ifidn __OUTPUT_FORMAT__,elf32
+section .note.GNU-stack noalloc noexec nowrite progbits
+%endif
+
diff -dur meka.orig/srcs/hq2x16.asm meka/srcs/hq2x16.asm
--- meka.orig/srcs/hq2x16.asm	2007-07-29 20:58:29.000000000 +0200
+++ meka/srcs/hq2x16.asm	2008-08-22 17:38:32.000000000 +0200
@@ -1936,3 +1936,7 @@
     dd ..@cross8, ..@flag0,  ..@flag0,  ..@flag0,
     dd ..@flag0,  ..@flag0,  ..@flag0,  ..@flag0
 
+%ifidn __OUTPUT_FORMAT__,elf32
+section .note.GNU-stack noalloc noexec nowrite progbits
+%endif
+
diff -dur meka.orig/srcs/hq2x32.asm meka/srcs/hq2x32.asm
--- meka.orig/srcs/hq2x32.asm	2007-07-29 20:58:29.000000000 +0200
+++ meka/srcs/hq2x32.asm	2008-08-22 17:39:56.000000000 +0200
@@ -1934,3 +1934,7 @@
     dd ..@cross8, ..@crossN, ..@crossN, ..@crossN,
     dd ..@crossN, ..@crossN, ..@crossN, ..@crossN
 
+%ifidn __OUTPUT_FORMAT__,elf32
+section .note.GNU-stack noalloc noexec nowrite progbits
+%endif
+
diff -dur meka.orig/srcs/mappersa.asm meka/srcs/mappersa.asm
--- meka.orig/srcs/mappersa.asm	2008-06-12 22:22:15.000000000 +0200
+++ meka/srcs/mappersa.asm	2008-08-22 17:38:55.000000000 +0200
@@ -173,3 +173,7 @@
 
 END
 
+%ifidn __OUTPUT_FORMAT__,elf32
+section .note.GNU-stack noalloc noexec nowrite progbits
+%endif
+
diff -dur meka.orig/srcs/videoasm.asm meka/srcs/videoasm.asm
--- meka.orig/srcs/videoasm.asm	2007-07-29 20:58:29.000000000 +0200
+++ meka/srcs/videoasm.asm	2008-08-22 17:56:49.000000000 +0200
@@ -495,3 +495,7 @@
 
 ;------------------------------------------------------------------------------
 
+%ifidn __OUTPUT_FORMAT__,elf32
+section .note.GNU-stack noalloc noexec nowrite progbits
+%endif
+
