diff -durN meka.orig/srcs/inputs.c meka/srcs/inputs.c
--- meka.orig/srcs/inputs.c	2014-10-02 14:21:42.000000000 +0200
+++ meka/srcs/inputs.c	2014-10-05 21:19:01.924936497 +0200
@@ -248,8 +248,8 @@
 void    Inputs_Switch_Joypad()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_JOYPAD);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_Joypad));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Digital));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_Joypad));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Digital));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
@@ -258,8 +258,8 @@
 void    Inputs_Switch_LightPhaser()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_LIGHTPHASER);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_LightPhaser));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Mouse));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_LightPhaser));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Mouse));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
@@ -268,9 +268,9 @@
 void    Inputs_Switch_PaddleControl()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_PADDLECONTROL);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_PaddleControl));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Mouse));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Digital_Unrecommended));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_PaddleControl));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Mouse));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Digital_Unrecommended));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
@@ -278,8 +278,8 @@
 void    Inputs_Switch_SportsPad()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_SPORTSPAD);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_SportsPad));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Mouse));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_SportsPad));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Mouse));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
@@ -287,8 +287,8 @@
 void    Inputs_Switch_GraphicBoard()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_GRAPHICBOARD);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_GraphicBoard));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Pen));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_GraphicBoard));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Pen));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
@@ -296,8 +296,8 @@
 void    Inputs_Switch_GraphicBoardV2()
 {
     Inputs_CFG_Peripheral_Change (PLAYER_1, INPUT_GRAPHICBOARD_V2);
-    Msg(MSGT_USER, Msg_Get(MSG_Inputs_GraphicBoardV2));
-    Msg(MSGT_USER_BOX, Msg_Get(MSG_Inputs_Play_Pen));
+    Msg(MSGT_USER, "%s", Msg_Get(MSG_Inputs_GraphicBoardV2));
+    Msg(MSGT_USER_BOX, "%s", Msg_Get(MSG_Inputs_Play_Pen));
     gui_menu_uncheck_range (menus_ID.inputs, 0, 5);
     gui_menu_check (menus_ID.inputs, Inputs.Peripheral[PLAYER_1]);
 }
